### PR TITLE
[BACKLOG-8963] Allow passthrough of parameters to the render engine (via type definition)

### PR DESCRIPTION
**Pull request by dcleao**
* visual.base.Model implementation
* ccc wrapper implementation, just missing blacklisting

@pentaho/millenniumfalcon please review.
@jheck123, please review the jsdoc changes.

### CODE REVIEW

**Comment by dcleao** [comment]
@pentaho/millenniumfalcon please review. This now has the code for all the tasks of the story.

**Comment by dcleao** [comment]
@jheck123 please review the changed doclets in the following files:
1. package-res/resources/web/pentaho/type/list.js
2. package-res/resources/web/pentaho/type/util.js (almost no changes; already reviewed in previews review, but was now moved to another file)
3. package-res/resources/web/pentaho/visual/base/_doc/spec/IModel.jsdoc
4. package-res/resources/web/pentaho/visual/base/model.js

File: package-res/resources/web/pentaho/visual/ccc/abstract/View.js [hunk 1]
```diff
@@ -623,7 +623,7 @@ define([
     },
 
     /**
-     * Adds a generic measure discriminator attribute info.
+     * Adds generic measure discriminator attribute infomation.
```
Comment (nantunes) [inline package-res/resources/web/pentaho/visual/ccc/abstract/View.js]: Should be "information".

Comment (dcleao) [inline package-res/resources/web/pentaho/visual/ccc/abstract/View.js]: Well spotted!

**Comment by nantunes** [comment]
If all lang check is complete, I'm ok with merging. @pentaho/millenniumfalcon any thing to add?

**Pull request merged**